src/zrender.ts

```diff
@@ -491,6 +504,8 @@ export interface ZRenderInitOpt {
     devicePixelRatio?: number
     width?: number | string // 10, 10px, 'auto'
     height?: number | string
+    darkMode?: 'auto' | 'light' | 'dark'
```

Comment:
Semantically, the `darkMode` option sounds like a boolean value, do we need to support `true/false` rather than `'dark'/'light'`?

Reply:
I was intentionly using `'light'/'dark'` instead of boolean values because I think it may be ambiguous whether `darkMode: true` means using dark mode whatever the system dark mode is, or use dark mode only when the system dark mode is true. I'm open to what you think.

Reply:
> I was intentionly using 'light'/'dark' instead of boolean values because I think it may be ambiguous whether darkMode: true means using dark mode whatever the system dark mode is, or use dark mode only when the system dark mode is true.

It looks clear enough to me:

- true - Always enable the dark mode.
- false - Never enable the dark mode even if the system is.
- 'auto' - follow the system preference.

This is how many apps do.

And if there is a must to avoid boolean value, 'colorScheme' (from CSS property [_**color-scheme**_](https://developer.mozilla.org/docs/Web/CSS/color-scheme)) may be a candidate for the name.